### ISSUE TITLE
Electron Cash --- SIGHASH FORKID error

### ISSUE THREAD

**Issue by cryptodavid** [fyookball/electrum#75]
When trying to send BCC/BCH I'm having an error :

The transaction was rejected by network rules. (16: mandatory-script-verify-flag-failed (Signature must use SIGHASH FORKID ))

**Comment by fyookball** [comment]
what version? what kind of wallet?  screenshot?

**Comment by dionbotha** [comment]
Hi fyookball, I am having the same issue... I have tried multiple servers. Some people say its because I have 2fa but I don't know how to get rid of it... The error comes straight after I google authenticate.
Any assistance would be greatly appreciated!

![capture1](https://user-images.githubusercontent.com/20134185/28997710-d694d802-79cf-11e7-90fa-13ce3518e0fe.PNG)
![capture2](https://user-images.githubusercontent.com/20134185/28997711-d69a5656-79cf-11e7-9e47-d29f1ca442c0.PNG)
![capture3](https://user-images.githubusercontent.com/20134185/28997709-d694cf60-79cf-11e7-9b1a-4f7b0e337c9e.PNG)
![capture4](https://user-images.githubusercontent.com/20134185/28997708-d69367d8-79cf-11e7-8b59-924452a25269.PNG)

**Comment by charleshsu168** [comment]
This is a known 2FA issue. See below from the 2FA multisig provider: TrustedCoin
------------------------------------------------
I'm the TrustedCoin admin. For a variety of reasons, we have no immediate plans to add support for BCC to our co-signing service. I suggest you modify the Trustedcoin plugin in the Electron Cash release to:

Not allow totally new 2FA wallets to be created
Allow existing 2FA wallets to be restored from seed only with the 2nd-factor disabled (i.e. restored with 2 keys in the wallet thus allowing unilateral spending).
When opening an existing 2FA wallet with only 1 of the 3 keys in it, a dialog should be shown informing the user they must restore from seed to regain access to their wallet.

**Comment by dionbotha** [comment]
For anyone out there with a similar issue not wanting to wait for an update,this is what I did to fix the issue.
- Got on a different computer that already had version 2.9.2 on it
- opened application and whatever wallet was already on that computer
- File -> New/Restore
- Gave a new wallet name
- Standard wallet
- I already have a seed
- entered the seed that I saved from when I first opened electron cash (2.9.0) on the first computer
It then told me I had 2fa on that wallet and asked if I want to disable -> yes.
I then could see the balance, chose the casharia network and sent the balance (with max tx fee for increased speed) to another wallet to test - success, no error!

1 confirmation so far but still not able to see it in the receiving wallet, will update later.
